internal/controllers/hub_cache_test.go

```
@@ -0,0 +1,230 @@
+		client:      mockClient,
+		scheme:      scheme,
+		entries:     map[string]*cachedHubEntry{},
+		entriesLock: &sync.Mutex{},
+		ttl:         DefaultHubCacheTTL,
+	}
+
+	ctx := context.Background()
+
+	// First call should hit the mock
+	_, err := cache.Get(ctx, "test-hub-id")
+	if err == nil {
```

golangci-lint (2.12.2)

[error] SA9003: empty branch

(staticcheck)